draft-ietf-taps-interface.md

```diff
@@ -563,9 +563,11 @@ the respective protocol has been selected.
 
 ## Transport Property Names {#property-names}
 
-Transport Properties are referred to by property names. These names are alphanumeric strings in which the following
-characters are allowed: lowercase letters `a-z`, uppercase letters `A-Z`,
-digits `0-9`, the hyphen `-`, and the underscore `_`. These names serve two purposes:
+Transport Properties are referred to by property names. These names are
+case-insensitive alphanumeric strings in which the following characters are allowed:
+letters `a-z` / `A-Z`, digits `0-9`, the hyphen `-`.
+The underscore `_` is only allowed at the first position.
```

Comment:
```suggestion
```

I don't understand the point of this rule, and it even contradicts what you say below about snake_case. The next sentence, "They must start with a letter or an underscore" seems good enough.

Reply:
Aha, I see...
```suggestion
The underscore `_` is only allowed at the first position (with the exception of transposing into snake_case, see below).
```